apps/admin/frontend/src/screens/tally_screen.tsx

```diff
@@ -116,6 +116,9 @@ export function TallyScreen(): JSX.Element | null {
               {hasAnyFiles ? (
                 <React.Fragment>
                   <tr>
+                    <TD as="th" narrow nowrap textAlign="right">
+                      #
+                    </TD>
```

Comment:
Why not just make this a `th`? Is it for styling?

Reply:
to be honest, I don't know, but I was just copying the approach of the other headers so as not to rock the boat.